AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Actions/FetchAuthorizationSession/InformSessionError.swift

```diff
@@ -21,7 +21,7 @@ struct InformSessionError: Action {
         logVerbose("\(#fileID) Starting execution", environment: environment)
         let event: AuthorizationEvent = switch error {
         case .service(let serviceError):
-            if isNotAuthorizedError(serviceError) {
+            if serviceError is AWSCognitoIdentityProvider.NotAuthorizedException {
```

Comment:
why a more restrictive check?

Reply:
1. Matching Android.
2. AWSCognitoIdentity `notAuthorizedException` should not be `sessionExpired` which could signal incorrectly for customers to push logout.